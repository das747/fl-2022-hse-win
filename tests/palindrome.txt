=> &S& -> @@
&S& -> @a@
&S& -> @b@
&S& -> @a@ &S& @a@
&S& -> @b@ &S& @b@
