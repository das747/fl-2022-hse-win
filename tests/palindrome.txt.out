Start non-terminal:
	'S'

Non-terminals:
	'S'

Terminals:
	''
	'a'
	'b'

Rules:
	'S' -> '' | 'a' | 'b' | 'a' 'S' 'a' | 'b' 'S' 'b'

Grammar is not in Chomsky normal form
